test/onnx/test_operators.py

```diff
@@ -122,6 +122,33 @@ def assertONNXRaisesRegex(self, err, reg, f, args, params=None, **kwargs):
         with self.assertRaisesRegex(err, reg):
             export_to_pbtxt(m, args, **kwargs)
 
+    def test_bad_symbolic_registration(self):
```

Comment:
nit: `test_utility_funs.py` would be a more suitable place for this kind of test.

Reply:
The similar test is 4 years old .. Back at the time there wasn't other test suites for different purposes.